src/components/BeaconsTable.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { render, screen, waitFor } from "@testing-library/react";
+import { testBeacons } from "../gateways/BeaconsGateway.testData";
+import { IBeaconsGateway } from "../gateways/IBeaconsGateway";
+import { BeaconsTable } from "./BeaconsTable";
+
+describe("<BeaconsTable>", () => {
+  let beaconsGatewayDouble: IBeaconsGateway;
+
+  beforeEach(() => {
+    beaconsGatewayDouble = {
+      getAllBeacons: jest.fn().mockResolvedValue(testBeacons),
+    };
+  });
+
+  it("renders a table", async () => {
+    render(<BeaconsTable beaconsGateway={beaconsGatewayDouble} />);
+
+    expect((await screen.findAllByRole("table")).length).toBeGreaterThan(1);
```

Comment:
🙈 not sure if this is really silly, but why should this be `> 1`? 

When I run the test, the `length` is 2 so I'm guessing it's something to do with the way the Material table is composed, but could `> 0` be like a less ✨ magic number-y ✨  way to checking the table is there? 🤔 Not sure though

Reply:
That's a totally fair comment!  Great catch.  Material _does_ use a `<table>` component for rendering the controls, so there are two tables per page.  I'll add a `data-testid="beacons-table"` to the `<table>` that matters 👍🏻

Reply:
> I'll add a data-testid="beacons-table" to the <table> that matters 👍🏻

Turns out I couldn't do this 🤦🏻  because material-table doesn't let you [override](https://material-table.com/#/docs/features/component-overriding) the `<table>` component...

>  could > 0 be like a less ✨ magic number-y ✨ way to checking the table is there? 🤔

... so I've gone with this instead 😅